<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Replace</class>
 <widget class="QDialog" name="Replace">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>240</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>490</width>
    <height>240</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>490</width>
    <height>240</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>DejaVu Sans</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Replace</string>
  </property>
  <widget class="QGroupBox" name="groupBox_startpos">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>90</y>
     <width>91</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>Start From</string>
   </property>
   <widget class="QRadioButton" name="radioButton_top">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Start search from top of the file or selection.</string>
    </property>
    <property name="text">
     <string>Top</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_bottom">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Start search from the bottom of the file or selection.</string>
    </property>
    <property name="text">
     <string>Bottom</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_replace">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>50</y>
     <width>101</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Replace with: </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_find">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>11</y>
     <width>41</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Find: </string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_mode">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>121</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>Search Mode</string>
   </property>
   <widget class="QRadioButton" name="radioButton_normal">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Search for a word or phrase.</string>
    </property>
    <property name="text">
     <string>Normal</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_reference">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Search using a regular expression.</string>
    </property>
    <property name="text">
     <string>References</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_findops">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>90</y>
     <width>121</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>Find In</string>
   </property>
   <widget class="QRadioButton" name="radioButton_current">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Find the text in the file currently selected and displayed in the main tab of the viewer.</string>
    </property>
    <property name="text">
     <string>Current File</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_selection">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>93</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Search the currently selected text in the viewer.</string>
    </property>
    <property name="text">
     <string>Selection</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>10</y>
     <width>101</width>
     <height>212</height>
    </rect>
   </property>
   <layout class="QVBoxLayout">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QPushButton" name="pushButton_findnext">
      <property name="toolTip">
       <string>Go to the next item found [F3].</string>
      </property>
      <property name="text">
       <string>Find Next</string>
      </property>
      <property name="shortcut">
       <string>F3</string>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_findprev">
      <property name="toolTip">
       <string>Go to the previous find [F4].</string>
      </property>
      <property name="text">
       <string>Find Prev</string>
      </property>
      <property name="shortcut">
       <string>F4</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_replace">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="toolTip">
       <string>Replace the currently selected match.</string>
      </property>
      <property name="text">
       <string>Replace</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_replaceall">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="toolTip">
       <string>Replace all matches.</string>
      </property>
      <property name="text">
       <string>Replace All</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_count">
      <property name="toolTip">
       <string>Displays how many items where found.</string>
      </property>
      <property name="text">
       <string>Count</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_findinfiles">
      <property name="text">
       <string>Find in Files</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_options">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>351</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Search Options</string>
   </property>
   <widget class="QCheckBox" name="checkBox_verbatim">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Search for exactly what is in the Find text box. Matches that are inside a word or otherwise partial finds are ignored.</string>
    </property>
    <property name="text">
     <string>Verbatim</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_case">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>20</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Case sensitive search.</string>
    </property>
    <property name="text">
     <string>Case</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_wrap">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Wrap</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_re">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>20</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>RegExp</string>
    </property>
   </widget>
  </widget>
  <widget class="QComboBox" name="comboBox_find">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>271</width>
     <height>29</height>
    </rect>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_replace">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>50</y>
     <width>221</width>
     <height>29</height>
    </rect>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
   <property name="maxVisibleItems">
    <number>20</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_findnext</sender>
   <signal>clicked()</signal>
   <receiver>Replace</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>383</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>137</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_findprev</sender>
   <signal>clicked()</signal>
   <receiver>Replace</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>383</x>
     <y>59</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>137</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_count</sender>
   <signal>clicked()</signal>
   <receiver>Replace</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>383</x>
     <y>161</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>137</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_findinfiles</sender>
   <signal>clicked()</signal>
   <receiver>Replace</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>385</x>
     <y>189</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>131</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_current</sender>
   <signal>clicked()</signal>
   <receiver>Replace</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>186</x>
     <y>111</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>131</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_selection</sender>
   <signal>clicked()</signal>
   <receiver>Replace</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>186</x>
     <y>141</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>131</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
